scalafix-tests/unit/src/test/scala/scalafix/tests/core/PatchSuite.scala

```diff
@@ -6,10 +6,15 @@ import java.nio.file.Paths
 
 import scala.meta._
 import scala.meta.tokens.Token.Ident
+import scala.meta.tokens.Token.classifiable
```

Comment:
:thinking: is that used? I guess so because we have scalafix guarding against unused imports?

Reply:
no, we don't apply scalafix on this project. But It worth adding it I think. Next  PR ^^

Reply:
Actually we apply scalafix on this project. It's an implicit import, and I guess it's hard to see that it's not used. Even Intellij doesn't see it's not used, or maybe it's used, but it's not a necessary import